### ISSUE TITLE
Explore Languages on homepage - hard to find/search for languages

### ISSUE THREAD

**Issue by missmatsuko** [exercism/exercism#4549]
On the ["Explore Languages" section of the homepage](https://exercism.io/#explore-languages), the languages are listed with their icons displayed. I'm not sure if there's an alternative listing that is more clear.

Listing languages by their icons poses a number of difficulties:
1. People may not recognize a language by its icon, especially new learners.
2. I'm unable to search for a language by its name on this page using the browser's text search (`ctrl/cmd + f`)
3. There doesn't seem to be an order to the languages (e.g. alphabetical), so I cannot find a language by looking around one that I do recognize.
4. I can see the name of the language by hovering over the icon, but this does not happen when the link is focused. So people who do not use a mouse have an extra disadvantage here.

**Comment by missmatsuko** [comment]
Found a more clear and searchable listing after I clicked one of the items and went to the 'Tracks' link in the breadcrumbs: https://exercism.io/tracks

**Comment by missmatsuko** [comment]
Had issues with this today when I was trying to find the Java track details from the homepage. I'm not that familiar with the logo, but I knew it was a coffee cup. There's 3 tracks on the homepage whose logos feature a coffee cup: Coffeescript, Erlang (LFE), Java. Very confusing!

**Comment by iHiD** [comment]
Hey :) So I can understand your use case, could you explain the reason for searching the page when you're logged out, rather than logging into your account where the tracks are listed alphabetically, searchable, and have details, please?

**Comment by missmatsuko** [comment]
Well, isn't every new user going to be doing that? I wanted to just read about the track details, I didn't really notice if I was logged in or not. I probably wouldn't log in until I had to do something that requires logging in, like starting a new course or submitting code.

**Comment by iHiD** [comment]
> Well, isn't every new user going to be doing that?

I don't know. Our landing page has a conversion rate of about 33%, which is pretty good, so I'm not sure this is blocking people from making the decision to sign up. And once you're in, the tracks UI is pretty clear. By adding more text to the landing page, I'm not sure that we improve the clarity of the page as much as add more noise.

> I probably wouldn't log in until I had to do something that requires logging in, like starting a new course or submitting code.

This is the bit that I find unusual, because most registered users stay logged in. I can't remember the last time I even saw the homepage personally. And if you were looking for the Java track, presumably you were looking to start it, which would have meant logging in.

I'm not saying you're wrong - this is just not in any way as obvious to me as it seems to be to you, so I'm trying to understand more. The honeycomb on the front page is really there as a visual thing to demonstrate the breadth of languages, not as a navigational aid. If we change it to that, then we should change the whole structure of that part, and I need more evidence to be comfortable investing time in doing that.

**Comment by missmatsuko** [comment]
Maybe there can be a link below the honeycomb cloud to the [full tracks listing view](https://exercism.io/tracks). It's much easier to get more info about the tracks from here, but I didn't see any way to access it except through breadcrumbs after I already click through to one of the courses. 

It's pretty hard to tell what tracks are available if you're not familiar with the logos, or you can't see them. Even if you're looking for a particular track, the track might display an uncommon/unofficial logo, so you still might not be able to recognize them visually. The mobile one has the logos and text, though. Can't the desktop one be laid out more similarly?

I guess I'm not using Exercism that much or something and it logged me out eventually? I don't think it's so unusual.

I just wanted to take a look at what the Java track is like. If there were only 2 challenges or it looked old/unmaintained, then I might not want to register? Or I might want to send a link to someone else without registering for it myself.

**Comment by sshine** [comment]
**TL;DR:** Adding a link below the honeycomb to the track listing page, as @missmatsuko suggests, would be an easy fix.

@missmatsuko wrote:
> Maybe there can be a link below the honeycomb cloud to the full tracks listing view.

I remember once about year ago I looked frantically after the Haskell track, not being able to find it because the honeycomb does not appear to be alphabetically sorted, as @missmatsuko points out.

Since I was new to Exercism, I didn't know that a searchable overview existed, so I spent probably a minute looking for the right icon. Since new users are presented to the honeycomb overview before the "full tracks listing" view, some may get caught up in this search like @missmatsuko and I did at one point.

@iHiD wrote:
> most registered users stay logged in. I can't remember the last time I even saw the homepage personally.

It's funny you should say that, because I generally have a hard time accessing the front page: Because I'm logged in, it redirects me to "/my/tracks" instead. (I've had to do that to investigate bug reports on those pages.) This creates a kind of blindness, and this issue and #4744 are examples of it.

@missmatsuko wrote:
> it logged me out eventually? I don't think it's so unusual.

Because I've recently been messing around with my Chrome (having at one point three different versions of it on my system), I have for some reason been getting logged out of Exercism quite often. So I don't think it's unusual.

**Comment by ErikSchierboom** [comment]
Hello 👋

With the launch of Exercism v3, we are **closing all issues in this repository** to help give us a clean slate to detect new problems. If this issue is still relevant to Exercism v3 (e.g. it's a feature that we haven't implemented in v3, or a bug that still exists), please reopen it and we will review it and post an update on it as soon as we get chance.

Thanks for helping make Exercism better, and we hope you enjoy v3 🙂
